3-projects/cloudbuild-tf-apply.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC
```

Comment:
I'm not sure we should be including a separate Cloud Build config for each module.

Reply:
I suppose we are expecting they would be bundled in the repo, but I get what you are saying. Perhaps we can add an issue to fix this as its the convention we are using for the other parts so far?

i.e we put the cloud build config in one place and copy it for each step

Reply:
I have added a issue for this here: https://github.com/terraform-google-modules/terraform-example-foundation/issues/11